src/Simulation/Simulators.Tests/TestProjects/TargetedExe/TargetedExe.csproj

```diff
@@ -1,17 +1,20 @@
-<Project Sdk="Microsoft.Quantum.Sdk">
+﻿<Project Sdk="Microsoft.Quantum.Sdk">
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <TargetFramework>net6.0</TargetFramework>
+    <QscVerbosity>detailed</QscVerbosity>
     <!-- we will provide our own -->
     <IncludeCSharpRuntime>false</IncludeCSharpRuntime>
     <IncludeQSharpCorePackages>false</IncludeQSharpCorePackages>
     <IncludeProviderPackages>false</IncludeProviderPackages>
-    <ExecutionTarget>honeywell.qpu</ExecutionTarget>
+    <ExecutionTarget>quantinuum.qpu</ExecutionTarget>
+    <TargetCapability>AdaptiveExecution</TargetCapability>
```

Comment:
Is the explicit TargetCapability needed here? With the new QDK behavior, this should be implied by the selection of the ExecutionTarget, right?